src/Core/Commander/Providers/NodeManager.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Generated On: 2016-09-09
+ * Class: NodeManager
+ * Description: Transition step between the
+ */
+
+ //https://github.com/Oslandia/workshop-3d-itowns/blob/master/2_analysis/5_3d_intersection.md
+
+function NodeManager() {
+	//Tab containing the different reprepresentation depending on the sse level for the features
+	// {levelMin, levelMax,representation}
+	this.levelSwitchTab = [ {min:  0, max:    4, type: 'point'},
+							{min:  4, max:   10, type: 'box'  },
+							{min: 10, max:   30, type: 'point'},
+							{min: 30, max: 1000, type: 'box'  }];
+}
+
+/**
+ * Check the feature type of the layer. It depends of the SSE, the type is kept inside the level switch tab.
+ * @param level: the layer current level
+ * @return type: if the type must be change return a string containing the new type, else return undefined
+ */
+NodeManager.prototype.checkType = function(node, quadtree, refinementCommandCancellationFn) {
+	var level = node.sse;
```

Comment:
As I removed every functions for the change of details, this file has been deleted.